src/ralph/assets/subscribers.py

```diff
@@ -0,0 +1,160 @@
+# -*- coding: utf-8 -*-
+import logging
+
+import pyhermes
+
+from django.conf import settings
+from django.contrib.auth import get_user_model
+
+from ralph.assets.models.assets import Environment, Service, ServiceEnvironment
+from ralph.assets.models.base import BaseObject
+
+logger = logging.getLogger(__name__)
+
+
+def _update_service_owners(service, business_owners, technical_owners):
+    """
+    Update business and technical owners for selected service.
+
+    Args:
+        service: Service
+        business_owners: dict of owners
+        technical_owners: dict of owners
+    """
+    business_owners = [
+        get_user_model().objects.get(username=i['username'])
```

Comment:
The question we have to ask here if we want to have a service without proper owners or we don't want such situation.

Consider sth like: `business_owners = get_user_model().objects.filter(username__in=[i['username'] for i in business_owners])`